usaspending_api/download/filestreaming/download_generation.py

```diff
@@ -31,7 +31,7 @@
 
```

Comment:
Did this file need to be changed?

Reply:
nope, it look like that was black doing it's thing

Reply:
Odd, I've been running black on the entire codebase for weeks and these haven't been affected

Reply:
i might have run it 3.5 instead of 3.7 or something? i just re-ran it on the same code and it didn't do anything so maybe my computer is just possessed